glide.yaml

```diff
@@ -0,0 +1,11 @@
+package: github.com/giantswarm/microendpoint
+import:
+- package: github.com/giantswarm/microerror
+  version: master
+- package: github.com/giantswarm/micrologger
+  version: master
+- package: github.com/go-kit/kit
+  version: v0.5.0
+  subpackages:
+  - log
+  - transport/http
```

Comment:
BTW I don't think you need to specify log package. It's specified by micrologger.